Gemfile

```diff
@@ -78,6 +79,7 @@ group :test do
 end
 
 group :development do
+  gem 'letter_opener'
```

Comment:
There's a few additions to the Gemfile from here down that don't seem to be related to this change so they should not be included in this pull request.